cmd/soroban-rpc/main.go

```diff
@@ -107,6 +107,20 @@ func main() {
 		},
 	}
 
+	versionCmd := &cobra.Command{
+		Use:   "version",
+		Short: "Print version information and exit",
+		Run: func(_ *cobra.Command, _ []string) {
+			if localConfig.CommitHash == "" {
+				fmt.Printf("soroban-rpc dev\n")
+			} else {
+				fmt.Printf("soroban-rpc %s (%s)\n", localConfig.Version, localConfig.CommitHash)
```

Comment:
Could we add the XDR version here too? Ideally the version of github.com/stellar/go/xdr and the version of stellar/stellar-xdr that was used to generate github.com/stellar/go/xdr.

Reply:
yes, that's my goal. I'll try to get there on the next PR.